_posts/2013-08-10-payrolls.markdown

```diff
@@ -40,6 +40,9 @@ layout: sidebar
     "employee_compensations": [
       {
         "employee_id": 1123581321345589,
+        "gross_pay": "2791.25",
+        "net_pay": "1953.31",
+        "payment_method":,
```

Comment:
Looks like this key is missing a value. Can you add "Direct Deposit" to the example?

Also, many of these fields won't show up unless processed is true. Can you change line 25 to true?